api/utils/misc/printer_test.go

```diff
@@ -1,4 +1,4 @@
-package common
+package misc
```

Comment:
Shouldn't this be `misc_test` like in `api/utils/misc/history_test.go`?

Reply:
It wasn't `_test` before to be able to use the package without import.

Reply:
Why would somebody want to import these tests?